package.json

```diff
@@ -41,17 +41,19 @@
     "prebuild:doc:kss": "run-s clean:doc:kss build:css:kss",
     "prepare": "yarn release",
     "prepush": "[[ $(git rev-parse --abbrev-ref HEAD) != 'gh-pages' ]] && yarn lint || true",
-    "release": "npm-run-all --parallel sprite palette && yarn transpile",
+    "release": "npm-run-all --parallel sprite palette && yarn transpile && yarn transpile:es5",
     "semantic-release": "semantic-release",
     "sprite": "scripts/make-icon-sprite.sh",
     "jest": "yarn test:jest",
     "screenshots": "node scripts/screenshots.js --screenshot-dir ./screenshots --styleguide-dir ./build/react",
     "test": "yarn test:jest",
     "test:jest": "jest",
-    "transpile:react": "env BABEL_ENV=transpilation babel react/ --out-dir transpiled/react --verbose",
-    "transpile:helpers": "babel helpers/ --out-dir transpiled/helpers --verbose",
-    "transpile": "yarn transpile:react && yarn transpile:helpers",
+    "transpile:react": "babel react/ --out-dir $BABEL_OUTPUT_DIR/react --verbose",
```

Comment:
@ptbrowne Do we need now to declare a $BABEL_OUTPUT_DIR var in our env?

Reply:
No, since the `transpile` method does that for you.

Reply:
Oh OK. I'm not used to `transpile`, I mainly use `transpile:react` directly to have less work to do when watching. Let's change my habit then

Reply:
See https://github.com/cozy/cozy-ui/pull/1368#discussion_r372209679